monkey/monkey_island/readme.md

```diff
@@ -132,3 +132,20 @@
 #### How to run
 
 1. From the `monkey` directory, run `python3.7 ./monkey_island.py`
+
+
+#### Troubleshooting
+
+While doing your first commit, there might be some python dependencies not installed on your system.
+The solution for resolving that is to create the `requirements.txt` and to install it
+both in `infection_monkey` and `monkey_island` directories.
```

Comment:
```suggestion
When committing your changes for the first time, you may encounter some errors thrown by the pre-commit hooks. This is most likely because some python dependencies are missing from your system.
To resolve this, use `pipenv` to create a `requirements.txt` for both the `infection_monkey/` and `monkey_island/` requirements and install it with `pip`.
```